dash-renderer/package.json

```
@@ -4,18 +4,19 @@
   "description": "render dash components in react",
   "main": "dash_renderer/dash_renderer.min.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint --quiet --fix .",
-    "lint:test": "./node_modules/.bin/eslint .",
+    "private::format.js-eslint": "eslint --quiet --fix .",
+    "private::format.js-prettier": "prettier --config .prettierrc --write \"src/**/*.js\"",
+    "private::lint.js-eslint": "./node_modules/.bin/eslint .",
```

```suggestion
    "private::lint.js-eslint": "eslint .",
```
Since you simplified the other one 😅

---

🙃

---

https://github.com/plotly/dash/pull/1151/commits/940648797e94e91e9816a59db99f9307b7583c6c